runtime/include/jvmti.h.m4

```diff
@@ -45,16 +45,20 @@ JNIEXPORT jint JNICALL Agent_OnAttach(JavaVM* vm, char *options, void *reserved)
 #define JVMTI_VERSION_1_1 0x30010100
 #define JVMTI_VERSION_1_2 0x30010200
 #define JVMTI_VERSION_1 (JVMTI_VERSION_1_0)
-#define JVMTI_VERSION_9  0x30090000
-#define JVMTI_VERSION_11 0x300b0000
 
-#define JVMTI_VERSION JVMTI_VERSION_11
+ifelse(eval(JAVA_SPEC_VERSION >= 9), 1, [#define JVMTI_VERSION_9  0x30090000
```

Comment:
Since 9 & 10 aren't supported, this could be 11.

Reply:
Or `> 8`

Reply:
sure, dropped the support of `JDK 9/10`.